### PR TITLE
update chainlink docs for Moonriver

### PR DESCRIPTION
**Pull request by eshaben**
Update the chainlink docs for Moonriver. This includes updating pre-existing Moonbase Alpha content to use the aggregator addresses (similarly to Moonriver)

### CODE REVIEW

File: builders/integrations/oracles/chainlink.md [hunk 1]
```diff
@@ -9,197 +9,110 @@ description: How to use request data from a Chainlink Oracle in your Moonbeam Et
...
+In a standard configuration, each price feed is updated by a decentralized Oracle network. Each Oracle node is rewarded for publishing the price data to the Aggregator contract. The Aggregator contract receives periodic data updates from the network of oracles and aggregates and stores the data on-chain so that consumers can easily fetch it. However, the information is only updated if a minimum number of responses from Oracle nodes are received (during an aggregation round).
 
-The end-user can retrieve price feeds with read-only operations via a Consumer contract, referencing the correct Aggregator interface (Proxy contract). The Proxy acts as a middleware to provide the Consumer with the most up-to-date Aggregator for a particular price feed.
+The end-user can retrieve price feeds with read-only operations via an Aggregator interface.
 
-![Price Feed Diagram](/images/builders/integrations/oracles/chainlink/chainlink-price-feed.png)
```
Comment (albertov19) [inline builders/integrations/oracles/chainlink.md:147]: Why did you remove this image :( haha I kind of like to have an image so it is not all text

File: snippets/text/chainlink/chainlink-brm.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ An Oracle node has a set of job IDs, where each corresponds to a task that can b
 
  - Oracle address: address of the contract deployed by the Oracle node
```
Comment (albertov19) [inline snippets/text/chainlink/chainlink-brm.md:3]: is this code snippet needed? I don't think we use it anymore

**Review by albertov19 — CHANGES_REQUESTED** [review]
Added some comments :)

File: builders/integrations/oracles/chainlink.md (hunk 1 above)
Comment (eshaben) [reply builders/integrations/oracles/chainlink.md:147]: I removed it because we don't talk about the consumer contract anymore so I thought it would just add more confusion

Comment (eshaben) [reply builders/integrations/oracles/chainlink.md:147]: I get what you're saying though, I did add 1 more screenshot to this page though if that counts for anything hahah

File: snippets/text/chainlink/chainlink-brm.md (hunk 2 above)
Comment (eshaben) [reply snippets/text/chainlink/chainlink-brm.md:3]: It's still under the Node Operators > Oracle Nodes > Chainlink section... should I remove the BRM stuff from there too? I thought it was more useful on that page 🤷‍♀️  but I would be happy to remove it if you think we should 🙂

File: builders/integrations/oracles/chainlink.md (hunk 1 above)
Comment (albertov19) [reply builders/integrations/oracles/chainlink.md:147]: Hahaha. But I think the consumer contract can still be used. I got this diagram from Chainlink official docs

File: snippets/text/chainlink/chainlink-brm.md (hunk 2 above)
Comment (albertov19) [reply snippets/text/chainlink/chainlink-brm.md:3]: Yeah i would go ahead.

File: builders/integrations/oracles/chainlink.md (hunk 1 above)
Comment (eshaben) [reply builders/integrations/oracles/chainlink.md:147]: ok I'll add it back in!

File: snippets/text/chainlink/chainlink-brm.md (hunk 2 above)
Comment (albertov19) [reply snippets/text/chainlink/chainlink-brm.md:3]: Actually I'll leave it because people can still do this part with BRM ;)

**Review by albertov19 — APPROVED** [review]
